api/pom.xml

```diff
@@ -218,61 +218,72 @@
-                    <windowtitle>${window.title}</windowtitle>
-                    <stylesheetfile>javax.servlet-api.css</stylesheetfile>
-                    <docfilessubdirs>true</docfilessubdirs>
-                    <bottom><![CDATA[Copyright &#169; {inceptionYear}&#x2013;{currentYear} <a href="https://eclipse.org">Eclipse Foundation</a>. Licensed under the <a href="{@docRoot}/doc-files/speclicense.html">Eclipse Foundation Specification License - v1.0</a>.]]></bottom>
-                    <groups>
-                        <group>
-                            <title>Servlet Documentation</title>
-                            <packages>javax.servlet</packages>
-                        </group>
-                    </groups>
-                    <tags>
-                        <tag>
-                            <name>implSpec</name>
-                            <placement>a</placement>
-                            <head>Implementation Requirements:</head>
-                        </tag>
-                        <tag>
-                            <name>param</name>
-                        </tag>
-                        <tag>
-                            <name>return</name>
-                        </tag>
-                        <tag>
-                            <name>throws</name>
-                        </tag>
-                        <tag>
-                            <name>since</name>
-                        </tag>
-                        <tag>
-                            <name>version</name>
-                        </tag>
-                        <tag>
-                            <name>serialData</name>
-                        </tag>
-                        <tag>
-                            <name>factory</name>
-                        </tag>
-                        <tag>
-                            <name>see</name>
-                        </tag>
-                    </tags>
-                </configuration>
+                <version>3.1.1</version>
                 <executions>
                     <execution>
+                        <id>attach-api-javadocs</id>
                         <goals>
                             <goal>jar</goal>
                         </goals>
+                        <configuration>
+                            <source>1.8</source>
+                            <additionalJOption>-Xdoclint:none</additionalJOption>
+                            <description>Jakarta Servlet API documentation</description>
+                            <doctitle>Jakarta Servlet API documentation</doctitle>
+                            <windowtitle>Jakarta Servlet API documentation</windowtitle>
+                            <header><![CDATA[<br>Jakarta Servlet API v${project.version}]]></header>
+                            <bottom><![CDATA[
+Comments to: <a href="mailto:servlet-dev@eclipse.org">servlet-dev@eclipse.org</a>.<br>
+Copyright &#169; 2019 Eclipse Foundation. All rights reserved.<br>
```

Comment:
why not use `{currentYear}` rather than 2019?

Reply:
Just `{currentYear}` wouldn't be valid when 2020 rolls around.